core/src/term/mod.rs

```diff
@@ -1377,7 +1377,7 @@ impl RichTerm {
     /// Erase recursively the positional information.
     ///
     /// It allows to use rust `Eq` trait to compare the values of the underlying terms.
-    //#[cfg(test)]
+    #[cfg(test)]
```

Comment:
I wonder why it was commented out, though. @vkleen do you use that in any of your libraries?

Reply:
I'm not using it in tf-ncl or json-schema-to-nickel.

Reply:
Git blame points to #683, so I think it was just an old oversight.

Reply:
It turns out it was necessary, because we're using it in the integration tests, and it apparently the integration tests build the nickel crate without `cfg(test)`

Reply:
I suspected something like that. Well, at lest let's get rid of the old comment